docs/configuration.md

```diff
@@ -82,17 +82,17 @@ Opposite to `skipStories`.
-| **`media`**             | _string_  | Emulates the given media for CSS media queries.                                                                                                           | _None_       |
-| **`fitWindow`**         | _boolean_ | Whether a view that exceeds the available browser window area should be scaled down to fit.                                                               | `chrome.*`   |
+| Name                                 | Type      | Description                                                                                                                                               | Targets      |
+| ------------------------------------ | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| **`target`**                         | _string_  | Target platform, possible values are `chrome.app`, `chrome.docker`, `ios.simulator`, `android.emulator`.                                                  | All          |
+| **`skipStories`**                    | _string_  | **DEPRECATED** Same as `loki.skipStories`, but applied to only this configuration.                                                                        | All          |
+| **`storiesFilter`**                  | _string_  | Same as `loki.storiesFilter`, but applied to only this configuration.                                                                                     | All          |
+| **`chromeSelector`**                 | _string_  | Same as `loki.chromeSelector`, but applied to only this configuration.                                                                                    | `chrome.*`   |
+| **`preset`**                         | _string_  | Predefined bundled configuration, possible values are `Retina Macbook Pro 15`, `iPhone 7`, `iPhone 5`, `Google Pixel`, `A4 Paper`, and `US Letter Paper`. | `chrome.*`   |
+| **`userAgent`**                      | _string_  | Custom user agent.                                                                                                                                        | `chrome.*`   |
+| **`width`**                          | _integer_ | Browser viewport width.                                                                                                                                   | `chrome.*`   |
+| **`height`**                         | _integer_ | Browser viewport height.                                                                                                                                  | `chrome.*`   |
+| **`disableAutomaticViewportHeight`** | _boolean_ | If the content goes below the viewport do not increase the height so that it fits.                                                                        | `chrome.*`   |
```

Comment:
Just noticed that the default value is wrong (also applies to some other values).

Reply:
Where is the default value?

Reply:
It's at the end of the line `chrome.*`.

Reply:
still not following

Reply:
No worries, I can fix all of them in my docs revamp.